### PR TITLE
moved export out of nav-links, added symbol

### PR DESCRIPTION
**Pull request by kenneth-miura**
## Motivation and Context
Addressing the section related to the export button on issue #1218.

## Your Changes
I altered `MasterTemplate.hs` to replace the Export element with an img. Then, I changed where it was defined so the img would be a direct child of `<nav class="row header">`. Then I added some styling so it would be in the top right corner.

**Description**:

**Type of change** (select all that apply):

- [x] Other: User Interface Change

## Testing

Clicked the new export button, and it still opens the export modal.
## Questions and Comments (if applicable)

I made Haskell changes that I'm not confident in.
## Checklist

- [x] I have performed a self-review of my own code.
- [x] I have verified that the CircleCI tests have
  passed.

### CODE REVIEW

**Review by david-yz-liu — COMMENTED** [review]
@kenneth-miura related to our discussion of accessibility today: it would be good to wrap the image in a button as well, and also set the `alt` text attribute of the button to "Export".

**Comment by kenneth-miura** [comment]
### Before
image

### After
image

File: app/MasterTemplate.hs [hunk 1]
```diff
@@ -43,15 +43,16 @@ header page =
...
-            then H.li $ H.a ! A.id "nav-export" $ "Export"
-            else ""
             H.li $ toLink "/generate" "Generate (beta)"
             -- H.li $ toLink "/timesearch" "Search"
             -- H.li $ toLink "/draw" "Draw"
             -- TODO: re-enable after handling new first-year courses
             -- H.li $ toLink "post" "Check My POSt!"
             H.li $ toLink "/about" "About"
+        if page `elem` ["graph", "grid"]
+        then H.button ! A.id "export" $ do
```
Comment (david-yz-liu) [inline app/MasterTemplate.hs:53]: The Haskell changes are indeed correct, but you should keep the old id (`nav-export`). It still fits, and in general it's best to avoid unnecessary changes.

File: style/app.scss [hunk 2]
```diff
@@ -218,9 +218,19 @@ body
   padding-bottom : 3px;
 }
 
-#nav-export
+#export
 {
-  cursor : pointer;
+  cursor: pointer;
```
Comment (david-yz-liu) [inline style/app.scss:223]: This rule is not necessary.

File: style/app.scss [hunk 3]
```diff
@@ -218,9 +218,19 @@ body
...
+  position: absolute;
```
Comment (david-yz-liu) [inline style/app.scss:224]: Similar to my comment on the other PR, you should try avoiding using absolute positioning.

This is a bit trickier to implement, especially after the nav bar changes in #1219 (which reminds, me you should update from master). A modern CSS approach is to use [**Flexbox**](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) on the nav bar (set `display: flex` and the `flex-direction: row`), and then have the three elements be the logo, the link `ul`, and the export button. After you get the general positioning right, you can use `text-align: right` on the button to make sure it's flush to the right.
